### PR TITLE
Conditions

### PR DESCRIPTION
**Pull request by josefkarasek**

### CODE REVIEW

File: pkg/operator/controller/machineconfig/controller.go [hunk 1]
```diff
@@ -91,7 +92,21 @@ func (r *MachineConfigReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 		return ctrl.Result{RequeueAfter: 1 * time.Minute}, err
 	}
 
-	return r.monitorProgress(ctx, req)
+	defer func() {
+		r.Log.Info("monitorPregress", "Status", r.CtrlConfig.Status)
```
[the remark this answers is not in the document]
Comment (josefkarasek) [reply pkg/operator/controller/machineconfig/controller.go:96]: Shouldn't be there at all! I used it for development, good catch!

**Review by sherine-k — APPROVED** [review]
/label qe-approved
/label docs-approved
/label px-approved

**Pull request merged**